### ISSUE TITLE
.gov-specific favicon?

### ISSUE THREAD

**Issue by konklone** [GSA/dotgov-home#13]
Right now it's the GSA agency logo. But I don't think that really represents the program very effectively, and it wouldn't distinguish it from gsa.gov in people's browser tabs. 

Other ideas? 

cc @h-m-f-t

**Comment by h-m-f-t** [comment]
...Was the favicon ever just the words "dotgov" or am I imagining it?

Here it is in Helvetica Neue with an approximation of "GSA blue" background:

![dotgov](https://user-images.githubusercontent.com/603901/28129920-0e676bda-66ea-11e7-8540-4c9c1cd2e669.png)
![image](https://user-images.githubusercontent.com/603901/28130118-b71a23c6-66ea-11e7-8bba-b4111d07ed4c.png)

though I think I like ".gov" better:
![dottedgov](https://user-images.githubusercontent.com/603901/28130215-0a26eab8-66eb-11e7-8832-47b0241d49ae.png)
![image](https://user-images.githubusercontent.com/603901/28130235-1e09e6f2-66eb-11e7-90c9-d712b78c04d4.png)

**Comment by konklone** [comment]
That's pretty good! I don't suppose you have that in SVG format? Even the larger PNG is a little blurry.

**Comment by konklone** [comment]
To be clear, the `.gov` one is the best, and I'm happy to use that ASAP. An SVG would be the ideal form, if you happen to have it, but a crisp large PNG will do too.

**Comment by h-m-f-t** [comment]
No one's ever accused me of being a graphic designer before, so take the attached under advisement. I used [Boxy SVG](https://boxy-svg.com/main.html) (thanks Boxy). 

Note that Helvetica Neue was not available, but there was an approximation (that I munged further to make the tip of the 'g' flat). It's looks about the same to my eyes when small.

[dottedgov.svg.zip](https://github.com/GSA/dotgov-home/files/1143646/dottedgov.svg.zip)
